cob_datapipeline/catalog_full_reindex_dag.py

```diff
@@ -231,6 +232,13 @@
     conn_id=SOLR_CLOUD.conn_id
 )
 
+SLACK_SUCCESS_POST = EmptyOperator(
+        task_id="slack_success_post",
+        dag=DAG,
+        on_success_callback=[slackpostonfail],
```

Comment:
@dkinzer Should this be slackpostonsuccess?

Reply:
oops, yup!